core/model/modx/processors/element/propertyset/updatefromelement.class.php

```diff
@@ -9,44 +10,39 @@
 
 class modPropertySetUpdateFromElementProcessor extends modPropertySetUpdateProcessor {
     public $languageTopics = array('propertyset', 'category', 'element');
-
-    /** @var modPropertySet|null */
-    public $object;
```

Comment:
@theboxer because modPropertySetUpdateFromElementProcessor extends modPropertySetUpdateProcessor which extends modObjectUpdateProcessor where this var already exists. https://github.com/Fi1osof/revolution/blob/patch-8/core/model/modx/modprocessor.class.php#L368